### DISCUSSION TITLE
run artisan commands before test

### DISCUSSION THREAD

**Discussion by emreakay** [pestphp/pest#434] [Q&A] [answered]
hi
i want tı run
artisan migrate:fresh 
artisan db:seed
 
commands on test db before start unit test

how can i do this?

**Comment by kossa** [comment]
I have exactly this issue, any answer?

**Comment by emreakay** [comment]
not yet

**Comment by fabio-ivona** [comment, marked as answer]
to refresh your database you can add this on top of your test file:

`uses(\Illuminate\Foundation\Testing\LazilyRefreshDatabase::class);`

and calling `seed()` in tests where you need to seed your database
